Automattic/vip-go-mu-plugins
# Two Factor 0.5.2

https://github.com/WordPress/two-factor/releases/tag/0.5.2

## To Test

- Verify log in flows work with and without 2fa
- Verify enabling / disabling 2fa works
- Verify admins without 2fa do not have privileged access
